Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/Editor/BuildInfo.cs

```diff
@@ -239,6 +243,24 @@ public virtual void ParseCommandLineArgs()
                         break;
                     case "-disableDebugging":
                         EditorUserBuildSettings.allowDebugging = false;
+                        Debug.LogWarning("This arg has been deprecated. use \"-allowDebugging false\" instead.");
+                        break;
+                    case "-allowDebugging":
+                        var value = arguments[++i];
```

Comment:
seems odd this isn't a switch

Reply:
it was before, but the logic was inverted. I figured being more explicit and deliberate was better for replacing this command